### DISCUSSION TITLE
Migrate deploy prisma_fmt_build_bg.wasm issue with AWS lambda function

### DISCUSSION THREAD

**Discussion by cuongdtt** [prisma/prisma#19603] [Q&A] [answered]
### Bug description

I am using `{ MigrateDeploy } from "@prisma/migrate"` to run deploy script on lambda

It's run very well in my local. But when I build and upload to lambda function it throw this error.
Issue: `ENOENT: no such file or directory, open src/prisma_fmt_build_bg.wasm`

_Do I need to upload prisma_fmt_build_bg file to lambda function?_

Step: 
- Build everything with Webpack
- Upload zip file to AWS lambda function

### Expected behavior

Migrate successfully

### Prisma information

```prisma
// Add your schema.prisma
```
simple schema with 1 table
```ts
// Add your code using Prisma Client
```

### Environment & setup

- OS: CentOS 8
- Database: PostgreSQL
- Node.js version: 18

### Prisma Version

```
**dev dependency**
prisma: 4.15.0

dependency
@prisma/migrate: 4.15.0
...

```

**Comment by ludralph** [comment]
Hi @dtcuong257 👋 

I found a similar issue to the one you are facing here

- https://github.com/prisma/prisma/issues/15881

There is a workaround specified in this comment linked here

- https://github.com/prisma/prisma/issues/15881#issuecomment-1289404724

If this answers your question, it would be great if you could **mark this Discussion as answered** to indicate that it has been resolved. 

Otherwise please let us know how else we can help you further or close the Discussion if it was resolved in some other way 🙏

**Reply by cuongdtt** [reply, marked as answer]
Thanks for you reply. It's not answered my question. 
I have some workaround by add prisma_fmt_build_bg.wasm and package.json file to make it work.
I'm not quire sure why :(

**Reply by ludralph** [reply]
Thanks for confirming what worked for you.
